src/Codec/Xlsx/Parser/Internal/PivotTable.hs

```diff
@@ -105,5 +106,5 @@ fillCacheFieldsFromRecords fields recs =
         then field {cfItems = mapMaybe recToCellValue recVals}
         else field
     recToCellValue (CacheText t) = Just $ CellText t
-    recToCellValue (CacheNumber n) = Just $ CellDouble n
+    recToCellValue (CacheNumber n) = Just $ CellDecimal (fromFloatDigits n)
```

Comment:
why not use Scientific in CacheNumber as well?

Reply:
Since I don't understand what CacheNumber is, I trust in your judgement here.

Reply:
Basically as far as I remember it's the same thing but for pivot table cache